README.md

```diff
@@ -31,19 +31,124 @@ You are welcome to send your plugin to our contrib by creating a Pull Request!
 | # | Plugin | Author | Feature |
 | :--- | :--- | :--- | :--- |
 | 1 | DingDong | [@wj-Mcat](https://github.com/wj-Mcat) | Reply `dong` if bot receives a `ding` message. |
+| 2 | DailyWords | [@markoxu](https://github.com/markoxu) | Say something everyday, like `Daily Words`. |
+| 3 | Rasa_Rest | [@markoxu](https://github.com/markoxu) | Rasa server channnel connector. |
+| 4 | Scheduler | [@wj-Mcat](https://github.com/wj-Mcat) | Say something everyday, like `Scheduler Words`. |
+| 5 | Room_inviter | [@markoxu](https://github.com/markoxu) | Invite peaple to room according keywords and rules then say hello words. |
+| 6 | Auto_reply | [@markoxu](https://github.com/markoxu) | AutoReply according keywords. |
+| 7 | Chat_history | [@markoxu](https://github.com/markoxu) | Save chat history with files and database. |
+| 8 | Github_webhook | [@wj-Mcat](https://github.com/wj-Mcat)| Github webhook, hook infomation to peaple or room when projects were updated. |
+| 9 | GitlabEvent | [@wj-Mcat](https://github.com/wj-Mcat) | Gitlab webhook, send gitlab event infomation to peaple or room when projects were updated. |
+| 10 | Finders | [@wj-Mcat](https://github.com/wj-Mcat) | Example code How to find room or contact, then do something. |
+| 12 | Use_two_plugins | [@xinyu3ru](https://github.com/xinyu3ru) | Example code How to use two or more plugins. |
```

Comment:
How did you get the author of this plugin ?  There are many mistake in author column.  Please correct it.